lib/spack/docs/index.rst

```diff
@@ -37,21 +37,24 @@ package:
 If you're new to spack and want to start using it, see :doc:`getting_started`,
 or refer to the full manual below.
 
------------------
-Table of Contents
------------------
-
 .. toctree::
    :maxdepth: 2
+   :caption: Documentation and Tutorial
 
    features
    getting_started
    basic_usage
+   workflows
+
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Reference Manual
+
    packaging_guide
    mirrors
    configuration
    developer_guide
-   case_studies
    command_index
    package_list
    API Docs <spack>
```

Comment:
I really like your proposal.

On Wed, Sep 7, 2016 at 5:34 PM, Adam J. Stewart <notifications@github.com>
wrote:

> In lib/spack/docs/index.rst:
>
> >     packaging_guide
> >     mirrors
> >     configuration
> >     developer_guide
> > -   case_studies
> >     command_index
> >     package_list
> >     API Docs <spack>
>
> In my mind, the distinction is pretty clear. Spack "Users" are people who
> only run Spack commands. Spack "Developers" are people who edit packages,
> core Spack libraries, or documentation, and then contribute that code back
> to Spack. I don't think the distinction between "Tutorial" and "Reference
> Manual" is any more clear.
>
> What about something like this?:
>
> .. toctree::
>    :maxdepth: 2
>    :caption: Tutorials
>
>    features
>    getting_started
>    basic_usage
>    workflows
>
> .. toctree::
>    :maxdepth: 2
>    :caption: Reference Manual
>
>    configuration
>    mirrors
>    package_list
>    command_index
>
> .. toctree::
>    :maxdepth: 2
>    :caption: Contributing to Spack
>
>    contribution_guide
>    packaging_guide
>    developer_guide
>    API Docs <spack>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/LLNL/spack/pull/1676/files/97cd009600fbdb2bbc654f6c0bdb1813436ef4c7#r77908363>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AB1cdx2PFevwYq16PFXWNub5A0KHuDH8ks5qny3PgaJpZM4JxOaa>
> .
>